mu/virtual_environment.py

```diff
@@ -545,8 +547,9 @@ def recreate(self):
         #
         # Now reinstall the original user packages
         #
-        logger.debug("About to reinstall user packages: %s", user_packages)
-        self.install_user_packages(user_packages)
+        if user_packages:

```

Comment:
Oh cool, this should fix https://github.com/mu-editor/mu/issues/1651.

Reply:
Thanks; I only just spotted it myself in passing. Didn't realise there was already an issue